src/app/rpmostree-dbus-helpers.c

```diff
@@ -479,12 +479,12 @@ on_transaction_progress (GDBusProxy *proxy,
   else if (g_strcmp0 (signal_name, "TaskEnd") == 0
            || g_strcmp0 (signal_name, "ProgressEnd") == 0)
     {
-      /* const gchar *message = NULL;
-         g_variant_get_child (parameters, 0, "&s", &message); */
+      const gchar *message = NULL;
+      g_variant_get_child (parameters, 0, "&s", &message);
```

Comment:
We should only do this for the `TaskEnd` case though right? `ProgressEnd` doesn't have a final msg.

Reply:
I think you're right in that this isn't used, but I believe the code supports it.  It might be useful at some point?

Reply:
It's not in the signature of the signal though, right?

```xml
    <signal name="TaskEnd">
      <arg name="text" type="s" direction="out"/>
    </signal>
...
    <signal name="ProgressEnd"/>
```

Reply:
Ah yes, good catch.  And that's also the source of the `g_warning()`s in the tests.